lib/Config/ServicesChecker.php

```diff
@@ -33,12 +33,15 @@ public function isMailPoetAPIKeyValid($displayErrorNotice = true, $forceCheck =
       || $mssKey['state'] == Bridge::KEY_INVALID
     ) {
       if ($displayErrorNotice) {
-        $error = Helpers::replaceLinkTags(
-          WPFunctions::get()->__('All sending is currently paused! Your key to send with MailPoet is invalid. [link]Visit MailPoet.com to purchase a key[/link]', 'mailpoet'),
-          'https://account.mailpoet.com?s=' . Subscriber::getTotalSubscribers(),
-          ['target' => '_blank']
-        );
-        WPNotice::displayError($error);
+        $error = '<h3>' . __('All sending is currently paused!', 'mailpoet') . '</h3>';
+        $error .= '<p>' . __('Your key to send with MailPoet is invalid.', 'mailpoet') . '</p>';
+        $error .= '<p><a '
+          . ' href="https://account.mailpoet.com?s=' . (Subscriber::getTotalSubscribers() + 1) . '"'
+          . ' class="button button-secondary" '
+          . ' target="_blank"'
+          . '>' . __('Visit MailPoet.com to purchase a key') . '</a></p>';
```

Comment:
I think this function call is missing the second argument 'mailpoet'.

Reply:
I thought phpstan can catch such errors.

Reply:
The second argument is optional. I guess we have to pass the correct domain https://mailpoet.slack.com/archives/G0A1L2KHV/p1571673372003200